playbooks/common/openshift-node/config.yml

```diff
@@ -215,6 +215,12 @@
                          | oo_collect('openshift.common.hostname') }}"
     openshift_node_vars: "{{ hostvars | oo_select_keys(groups['oo_nodes_to_config']) }}"
   pre_tasks:
-
+  - name: Wait for master to become available before proceeding
+    wait_for:
+      host: "{{ hostvars[groups.oo_first_master.0].openshift.common.ip }}"
+      port: 8443
```

Comment:
We should pull the api port from facts as well, otherwise this will fail when overriding the ports.